app/controllers/uploads_controller.rb

```diff
@@ -46,11 +48,32 @@ def show
     redirect_to(root_path) and return unless @upload
     return unless @upload.completed?
 
-    measure_ids = @upload.qrda_files.collect{ |file| file.get_measure_ids }.flatten.uniq
-
     @bundle = BUNDLES[@upload.year]
-    @measures = @bundle ? @bundle.measures.in(hqmf_id: measure_ids) : []
 
     @files = @upload.qrda_files
   end
+
+  def shouldSwitchHighlight (dataCriteriaKey, populationCriteriaKey, specifics, rationale)
```

Comment:
There's a number of a rubocop errors in this function. Rather than call out each one here, you can see them if you run `rubocop <filename>` (don't just run rubocop because there's approximately 46872384572308 rubocop errors in the project in total).